src/error.rs

```diff
@@ -1150,6 +1155,13 @@ impl ToDiagnostic<FileId> for ParseError {
                 Diagnostic::error()
                     .with_message(format!("{} parse error: {}", format, msg))
                     .with_labels(labels)
+            },
+            ParseError::UnboundTypeVariables(idents) => {
+                Diagnostic::error()
+                    .with_message(format!("Unbound type variable(s): {}", idents.into_iter()
+                        .map(|x| format!("\"{}\"", x))
```

Comment:
```suggestion
                        .map(|x| format!("`{}`", x))
```
It is currently customary to report identifiers, types etc. in error message quoted by `, to differentiate them from strings.